packages/twenty-front/project-structure.cjs

```diff
@@ -0,0 +1,472 @@
+/* eslint-disable project-structure/folder-structure */
+// @ts-check
+
+const { default: src } = require('afterframe');
+const { createFolderStructure } = require('eslint-plugin-project-structure');
+
+module.exports = createFolderStructure({
+  longPathsInfo: {
+    root: './packages/twenty-front',
+    mode: 'warn',
+  },
+  structure: [
+    // Allow any files in the root of your project, like package.json, eslint.config.mjs, etc.
+    // You can add rules for them separately.
+    // You can also add exceptions like this: "(?!folderStructure)*".
+    { name: '*' },
+    {
+      name: 'packages',
+      children: [
+        {
+          name: 'twenty-front',
+          children: [
```

Comment:
i'm not sure we should talk about the files out of the src/modules folder for now, is there a way not taking them into account at all?